docs/conf.py

```diff
@@ -130,6 +139,10 @@
         for line in fin:
             new_line = re.sub(
                 '<.*examples_gallery.*>', '<examples_gallery>', line)
+            new_line = re.sub(
+                '<.*community/index.*>', '<community/index>', new_line)
+            new_line = re.sub(
+                '<.*help/index.*>', '<help/index>', new_line)
```

Comment:
```suggestion
            new_line = line
            for version in ['latest', 'stable']:
                pattern = fr'(https://pyfar-gallery\.readthedocs\.io/en/{version}/(.*?)\.html)'
                match = re.search(pattern, new_line)
                if match:
                    new_line = re.sub(pattern, match.group(2), new_line)
```
this will generally rename all readthedocs links to the local path, works for stable and latest links